includes/extra_configures/enable_error_logging.php

```diff
@@ -21,8 +21,9 @@ function zen_debug_error_handler($errno, $errstr, $errfile, $errline)
         return;
     }
 
+    static $last_log_suffix;
     if (!isset($last_log_suffix)) {
-        static $last_log_suffix = '.log';
```

Comment:
As otherwise described in the chain of this conversation, an appropriate modification here to address just the use of `static` and there being only one initial action to be taken when this method is called would be to simply remove the `if` statement and leave the line to initially declare the `static` variable to whatever suffix is to initially be used.  Upon each call of this method during the current session, the last known value of the variable will be retained and ***not reassigned*** to the initial variable.

By placing the assignment within the if statement allows for other/additional action to be taken the first time the variable is declared (or if it somehow becomes `null` in iterative processing).

Although at the time of suggesting this edit I apparently did not fully understand that the `static` variable would not be reassigned to the original value, I *am* ok with it still being in the `if` statement as proposed here if this commit remains or is resurrected. This would allow others to "easily" incorporate additional first time actions into this section rather than having to incorporate/add it later... Just saying. :)

Reply:
I'm fine with relocating the `static` declaration to outside the `if` statement.